stellarphot/core.py

```diff
@@ -30,6 +30,36 @@
 # https://docs.gammapy.org/dev/_modules/gammapy/analysis/config.html
 
 
+class UnitType(Unit):
+    # Validator for Unit type
+    @classmethod
+    def __get_validators__(cls):
+        yield cls.validate
+
+    @classmethod
+    def validate(cls, v):
```

Comment:
What is this validate doing?  It just returns the Unit cast version of v?  Does it actually validate anything?

Reply:
If the creation of the unit fails (e.g. Unit("nonsense")) then it generates a `ValidationError`